### PR TITLE
[p5] updated to latest (V1.4.0)

### PR DESCRIPTION
**Pull request by Gaweph**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by Gaweph** [comment]
Fixes the problem identified here (p5 typings out of date):
[@definitelytyped/dtslint/dt.json](https://github.com/Gaweph/p5-typescript-starter/issues/28)

**Review by Zalastax — APPROVED** [review]
Looks good and uses the preferred tooling for updating p5.js types so that's great!

**Comment by Gaweph** [comment]
@typescript-bot is there anything more that needs to be done to get this PR approved?

**Comment by Gaweph** [comment]
Ready to merge

File: types/p5/tslint.json [hunk 1]
```diff
@@ -1,6 +1,12 @@
 {
     "extends": "@definitelytyped/dtslint/dt.json",
     "rules": {
+        "file-name-casing": false,
         "unified-signatures": false
-    }
+    },
+    "cliOptions": {
```
Comment (gabritto) [inline types/p5/tslint.json:7]: Why are those changes needed?

**Review by gabritto — APPROVED** [review]

**Comment by Gaweph** [comment]
Ready to merge
